core/src/style/pointStyle.cpp

```diff
@@ -115,10 +117,16 @@ PointStyle::Parameters PointStyle::applyRule(const DrawRule& _rule, const Proper
 void PointStyle::pushQuad(std::vector<Label::Vertex>& _vertices, const glm::vec2& _size, const glm::vec2& _uvBL,
                           const glm::vec2& _uvTR, unsigned int _color, float _extrudeScale) const {
 
-    _vertices.push_back({{    0.0,       0.0}, {_uvBL.x, _uvTR.y}, {-1.f,  1.f, _extrudeScale}, _color});
-    _vertices.push_back({{_size.x,       0.0}, {_uvTR.x, _uvTR.y}, { 1.f,  1.f, _extrudeScale}, _color});
-    _vertices.push_back({{    0.0,  -_size.y}, {_uvBL.x, _uvBL.y}, {-1.f, -1.f, _extrudeScale}, _color});
-    _vertices.push_back({{_size.x,  -_size.y}, {_uvTR.x, _uvBL.y}, { 1.f, -1.f, _extrudeScale}, _color});
+    float es = _extrudeScale;
+
+    // Attribute will be normalized - scale to max short;
+    glm::vec2 uvTR = _uvTR * texture_scale;
+    glm::vec2 uvBL = _uvBL * texture_scale;
```

Comment:
It would just be superfluous to use it for text as the scaling must be done in the shader anyway - so it would just be an additional multiplication when using automatic normalization (I assume that for automatic normalization the values are transformed during upload to a float representation). One may also loose a bit of precision during the back and forth transformation

If we decide that uv coordinates must be in the range -1..1 we can use normalization for the other styles